packages/server/src/schema/operations.ts

```diff
@@ -0,0 +1,14 @@
+import { Resolver } from './resolvers';
+import * as resolvers from './resolvers';
+
+export const createSimulation = createResolve(resolvers.createSimulation);
+
+export const given = createResolve(resolvers.given);
+
+function createResolve(resolver: Resolver<unknown, unknown>) {
```

Comment:
should it not be `createResolver`

Reply:
I renamed it to `uncover()` since the only reason is to strip the type information.